### PR TITLE
Update AlertManager SMTP TLS configuration options

### PR DESCRIPTION
**Pull request by sgissi**
@juliusv  Update AlertManager configuration options related to [https://github.com/prometheus/alertmanager/pull/512]

### CODE REVIEW

File: content/docs/alerting/configuration.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ global:
   [ smtp_auth_username: <string> ]
   [ smtp_auth_password: <string> ]
   [ smtp_auth_secret: <string> ]
+  # The default SMTP TLS requirement
```
Comment (juliusv) [inline content/docs/alerting/configuration.md]: Period at the end.

**Comment by juliusv** [comment]
:+1: besides nit

**Comment by sgissi** [comment]
Thanks, added the missing trailing period.

@fabxc @juliusv The docs are published only after a new release of AlertManager? Just to make sure users don't try to use a feature that only exists in master.

**Comment by juliusv** [comment]
@sgissi Ah, good point! For Prometheus releases, we have the `next-release` doc branch, but I don't believe we have one for the Alertmanager yet. Maybe best to let this sit here then until @fabxc or @brancz bake the next Alertmanager release.

**Comment by sgissi** [comment]
With the new AM released we should merge this. Should I mention in the comment line that it applies to AlertManager >= 0.5.0 to avoid luring the users of the packaged versions into using an invalid configuration?

**Comment by brancz** [comment]
As far as I understand the typical process is to have the docs reflect the latest released stable version, but I think there are some mentions of previous versions of Prometheus as well. Wdyt about this @juliusv @fabxc @brian-brazil ?

**Comment by fabxc** [comment]
Yea, we generally just refer to the latest stable version.
We hopefully figure out a general approach to versioned docs eventually. Mentioning exceptions across versions is a nightmare we don't want to get into.

**Pull request merged**